CMakeLists.txt

```diff
@@ -2,12 +2,6 @@ cmake_minimum_required(VERSION 3.4...3.20 FATAL_ERROR)
 
 project(GLFW VERSION 3.4.0 LANGUAGES C)
 
-set(CMAKE_LEGACY_CYGWIN_WIN32 OFF)
```

Comment:
"On CYGWIN, the undocumented “CMAKE_LEGACY_CYGWIN_WIN32” mode for compatibility with CMake versions older than 2.8.4 has been removed." - https://discourse.cmake.org/t/cmake-3-26-0-rc1-is-ready-for-testing/7394